### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by ssrlive**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ use std::sync::Arc;
 //Must be run as Administrator because we create network adapters
 //Load the wintun dll file so that we can call the underlying C functions
 //Unsafe because we are loading an arbitrary dll file
-let wintun = wintun::load_from_path("path/to/wintun.dll")
+let wintun = unsafe { wintun::load_from_path("path/to/wintun.dll") }
```
Comment (TroyNeubauer) [inline README.md:28]: Good catch with the needed unsafe

File: examples/udp-echo.rs [hunk 2]
```diff
@@ -236,15 +237,17 @@ fn extract_udp_packet(packet: &[u8]) -> Result<NaiveUdpPacket, wintun::Error> {
                     return Ok(udp_packet);
                 }
                 _ => {
-                    log::trace!("{:?} src={}, dst={}", protocol, src_addr, dst_addr);
+                    info = format!("{:?} src={}, dst={}", protocol, src_addr, dst_addr);
+                    log::trace!("{}", info);
```
Comment (TroyNeubauer) [inline examples/udp-echo.rs]: I would perfer to not log here and return the error directly. That way the caller can decide what to do with it and by default we remove non UDP logs which we are less likely to want in this example.

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -41,3 +41,7 @@ jobs:
         with:
           command: clippy
           args: --all-features -- -D warnings
+      - uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --examples --all-features
```
Comment (TroyNeubauer) [inline .github/workflows/main.yml:47]: Nice

**Review by TroyNeubauer — CHANGES_REQUESTED** [review]
I like the cleanup, but I'm not sold on the purpose of the `build.rs`. 

In its current form, the `build.rs` copies the dll for the current platform into the tests directory so we can do: `wintun::load_from_path("wintun.dll")` instead of `wintun::load_from_path("wintun/bin/amd64/wintun.dll")`. 

I don't believe the extra complexity of build.rs is worth making one line simpler in each of the examples. I would prefer copying the `get_wintun_bin_relative_path` function to each example and using that, or inlining the function to include the platform branches where `wintun::load_from_path` is called.

File: examples/udp-echo.rs (hunk 2 above)
Comment (ssrlive) [reply examples/udp-echo.rs]: OK

**Comment by ssrlive** [comment]
I don't think the complexity of the build.rs script affects the behavior of the main code, it just adds to user convenience. Ordinary users will even simply ignore this script. If you place this function inside the project, this also increases the redundancy of the code, that is, the transactional secondary code becomes part of the project.

**Comment by TroyNeubauer** [comment]
Given the recent staging of malware on crates.io which utilized `build.rs` to execute malicious code on developer's machines, users will be looking at `build.rs` with even more scrutiny, they simply cannot ignore it: https://blog.phylum.io/rust-malware-staged-on-crates-io/. Because there is no strong reason for us to have a `build.rs` besides simplifying the example code, I prefer to avoid making users nervous by not having one. 

Also most of the time this crate is compiled, it will be as a dependency where no examples will be run. However the dll will still be copied on every build.

I agree that copy pasting the function does increase the redundancy, but to me it is better to keep things explicit and redundant than add the complexity of `build.rs` and additional stress for users. The bottom of the file is a perfect place for a secondary duplicated function.

**Comment by ssrlive** [comment]
OK . here the changes.
image

**Comment by ssrlive** [comment]
Please use Squash merge to hide build.rs file
image

**Comment by TroyNeubauer** [comment]
Thanks for your perseverance! Looks good

**Review by TroyNeubauer — APPROVED** [review]

**Pull request merged**